internal/providers/terraform/aws/data_transfer.go

```diff
@@ -14,8 +16,6 @@ var regionMapping = map[string]string{
 	"us-west-1":       "US West (N. California)",
 	"us-west-2":       "US West (Oregon)",
 	"us-west-2-lax-1": "US West (Los Angeles)",
-	"us-iso-east-1":   "US ISO East",
-	"us-isob-east-1":  "US ISOB East (Ohio)",
```

Comment:
These are are secret regions, only used by US secret services :D so ok for them to be removed.